zipline/pipeline/domain.py

```diff
@@ -100,27 +123,53 @@ class EquityCalendarDomain(Domain):
         ISO-3166 two-letter country code of the domain
     calendar_name : str
         Name of the calendar, to be looked by by trading_calendar.get_calendar.
+    data_query_offset : datetime.timedelta
+         The offset from market open when data should no longer be considered
+         available for a session.
```

Comment:
An example might be helpful here, e.g. "For example, a data_query_offset of -timedelta(minutes=45) signifies that data must have been available 45 minutes prior to market open for it to appear in pipeline inputs for a given day."